proposals/compatibility-mode.md

```diff
@@ -95,6 +95,30 @@ Use of the `sample_mask` builtin would cause a validation error at pipeline crea
 
 **Justification**: OpenGL ES 3.1 does not support `gl_SampleMask` or `gl_SampleMaskIn`.
 
+### 8. Lower limits.
+
+The differences in limits between compatibility mode and standard WebGPU
+are as follows
+
+
+| limit                             | compat  | standard  |
+| :-------------------------------- | ------: | --------: |
+| maxColorAttachments               |       4 |         8 |
+| maxComputeInvocationsPerWorkgroup |     128 |       256 |
+| maxComputeWorkgroupSizeX          |     128 |       256 |
+| maxComputeWorkgroupSizeY          |     128 |       256 |
+| maxInterStageShaderVariables      |      15 |        16 |
+| maxStorageBuffersPerShaderStage   |       4 |         8 |
+| maxTextureDimension1D             |    4096 |      8192 |
+| maxTextureDimension2D             |    4096 |      8192 |
+| maxTextureDimension3D             |    1024 |      2048 |
+| maxUniformBufferBindingSize       |   16384 |     65536 |
+| maxVertexAttributes (see below)   |   14-16 |        16 |
```

Comment:
The actual value of this limit would be 16 then. I think this table should say that for clarity. (or "16*" / "16 <sup>a</sup>")

Reply:
updated